src/styles/components/section-title.scss

```diff
@@ -0,0 +1,32 @@
+.section-title {
+  font-size: 32px;
+  line-height: 32px;
+  letter-spacing: -1.5px;
+  text-align: center;
+
+  @include on-tablet {
+    font-size: 48px;
+    line-height: 48px;
+  }
+  &__now-on-view {
+    text-align: center;
+
+    margin-bottom: 8px;
+  }
+
+  &__now-annonce {
+    font-size: initial;
+    line-height: 27px;
+    letter-spacing: 0;
+
+    margin-bottom: 48px;
+  }
+  &-online {
+    text-align: center;
+
+    margin-bottom: 40px;
+  }
+  &-contacts {
+    margin-bottom: 48px;
+  }
```

Comment:
Also in this place, make online and contacts modifiers or write full selector name